src/modules/mavlink/mavlink_receiver.h

```diff
@@ -189,90 +191,101 @@ class MavlinkReceiver
 	 */
 	int decode_switch_pos_n(uint16_t buttons, unsigned sw);
 
-	bool	evaluate_target_ok(int command, int target_system, int target_component);
+	bool evaluate_target_ok(int command, int target_system, int target_component);
 
 	void send_flight_information();
 
 	void send_storage_information(int storage_id);
 
-	Mavlink	*_mavlink;
-
-	MavlinkMissionManager		_mission_manager;
-	MavlinkParametersManager	_parameters_manager;
-	MavlinkFTP			_mavlink_ftp;
-	MavlinkLogHandler		_mavlink_log_handler;
-	MavlinkTimesync		_mavlink_timesync;
-
-	mavlink_status_t _status; ///< receiver status, used for mavlink_parse_char()
-	struct vehicle_attitude_s _att;
-	struct vehicle_local_position_s _hil_local_pos;
-	struct vehicle_land_detected_s _hil_land_detector;
-	struct vehicle_control_mode_s _control_mode;
-	orb_advert_t _global_pos_pub;
-	orb_advert_t _local_pos_pub;
-	orb_advert_t _attitude_pub;
-	orb_advert_t _gps_pub;
-	orb_advert_t _gyro_pub;
-	orb_advert_t _accel_pub;
-	orb_advert_t _mag_pub;
-	orb_advert_t _baro_pub;
-	orb_advert_t _airspeed_pub;
-	orb_advert_t _battery_pub;
-	orb_advert_t _cmd_pub;
-	orb_advert_t _flow_pub;
-	orb_advert_t _hil_distance_sensor_pub;
-	orb_advert_t _flow_distance_sensor_pub;
-	orb_advert_t _distance_sensor_pub;
-	orb_advert_t _offboard_control_mode_pub;
-	orb_advert_t _actuator_controls_pubs[4];
-	orb_advert_t _att_sp_pub;
-	orb_advert_t _rates_sp_pub;
-	orb_advert_t _pos_sp_triplet_pub;
-	orb_advert_t _mocap_odometry_pub;
-	orb_advert_t _visual_odometry_pub;
-	orb_advert_t _radio_status_pub;
-	orb_advert_t _ping_pub;
-	orb_advert_t _rc_pub;
-	orb_advert_t _manual_pub;
-	orb_advert_t _obstacle_distance_pub;
-	orb_advert_t _trajectory_waypoint_pub;
-	orb_advert_t _land_detector_pub;
-	orb_advert_t _follow_target_pub;
-	orb_advert_t _landing_target_pose_pub;
-	orb_advert_t _transponder_report_pub;
-	orb_advert_t _collision_report_pub;
-	orb_advert_t _debug_key_value_pub;
-	orb_advert_t _debug_value_pub;
-	orb_advert_t _debug_vect_pub;
-	orb_advert_t _debug_array_pub;
+	Mavlink *_mavlink;
+	MavlinkFTP               _mavlink_ftp;
+	MavlinkLogHandler        _mavlink_log_handler;
+	MavlinkTimesync          _mavlink_timesync;
+	MavlinkMissionManager    _mission_manager;
+	MavlinkParametersManager _parameters_manager;
+
+	mavlink_status_t _status                          = {}; ///< receiver status, used for mavlink_parse_char()
+	struct vehicle_attitude_s _att                    = {};
+	struct vehicle_local_position_s _hil_local_pos    = {};
+	struct vehicle_land_detected_s _hil_land_detector = {};
+	struct vehicle_control_mode_s _control_mode       = {};
+
+	orb_advert_t _actuator_controls_pubs[4] = {nullptr, nullptr, nullptr, nullptr};
+
+	orb_advert_t _accel_pub                 = nullptr;
```

Comment:
This is nitpicky, but what about doing this with uniform initialization? It looks a bit weird at first, but there are advantages.
I've been trying to do is consistently across the codebase.

Reply:
Thanks @dagar !  Done and pushed up!